feature_engine/wrappers/wrappers.py

```diff
@@ -144,6 +144,45 @@ class SklearnTransformerWrapper(BaseEstimator, TransformerMixin):
     See Also
     --------
     sklearn.compose.ColumnTransformer
+
+    Examples
+    --------
+
+    >>> import pandas as pd
+    >>> from feature_engine.wrappers import SklearnTransformerWrapper
+    >>> from sklearn.preprocessing import StandardScaler
+    >>> X = pd.DataFrame(dict(x1 = [1,2,3], x2 = [4,5,6]))
+    >>> skw = SklearnTransformerWrapper(StandardScaler())
+    >>> skw.fit(X)
+    >>> skw.transform(X)
+             x1        x2
+    0 -1.224745 -1.224745
+    1  0.000000  0.000000
+    2  1.224745  1.224745
+
+    >>> import pandas as pd
+    >>> from feature_engine.wrappers import SklearnTransformerWrapper
+    >>> from sklearn.preprocessing import OneHotEncoder
+    >>> X = pd.DataFrame(dict(x1 = ["a","b","c"], x2 = [4,5,6]))
+    >>> skw = SklearnTransformerWrapper(OneHotEncoder(sparse = False),
+    >>>                                 variables = "x1")
+    >>> skw.fit(X)
+    >>> skw.transform(X)
+       x2  x1_a  x1_b  x1_c
+    0   4   1.0   0.0   0.0
+    1   5   0.0   1.0   0.0
+    2   6   0.0   0.0   1.0
+
+    >>> import pandas as pd
+    >>> from feature_engine.wrappers import SklearnTransformerWrapper
+    >>> from sklearn.preprocessing import PolynomialFeatures
+    >>> skw = SklearnTransformerWrapper(PolynomialFeatures(include_bias = False))
```

Comment:
the examples are meant for the user to be able to copy and paste and reproduce, so we need to regenerate X here. Also, it'd be nicer if we added 1 additional numerical variable to be able to make more combinations. But do keep the categorical one, because its the one that justifies the use of the wrapper in this example.